app/views/contact/foi/new.html.erb

```diff
@@ -8,7 +8,12 @@
   <%= form_for :contact, url: contact_foi_path, as: :post, authenticity_token: false, html: { class: "contact-form" } do |f| %>
     <%= hidden_field_tag 'foi[url]', Plek.new.website_root + contact_foi_path %>
 
-    <input type="text" id="val" name="foi[val]" style="display:none"/>
+    <%# Honeypot field intended to reduce spam %>
+    <div class="val" aria-hidden="true">
```

Comment:
The class `visually-hidden` does the opposite of what `aria-hidden=true` does. The former removes it visually but it will still be read by screen readers while the latter means it will **not** be read by screen readers but can be visually shown (or not, depending on what else there is hiding or showing it).